### DISCUSSION TITLE
Can you please help with settings?

### DISCUSSION THREAD

**Discussion by Xaos-biker** [Chia-Network/chia-blockchain#2212] [General]
Hello everyone. Pleasу help me with optimal settings for fast plotting on my eqiment:

10pcs - 18Tb Seagate HDD
1pcs - 2Tb NVME SSD 3500/3000
1pcs - 120GB SSD for Windows or other OC
4pcs*8 - in total 32GB DDR4 2666MHz
1pcs - Ryzen 7 2700 Tray 8x3,2 GHz (Turbo Boost 4,1 GHz)
1pcs - Motherboard ASROCK Fatal1ty B450
1pcs - 600Wat Chieftec

Can you please help me with settings for faster plotting (Windows will be better)? And what prefer operating system i must to use for fastter start mining?

**Comment by Xaos-biker** [comment]
plotting time more than 24 hours (for 6 plots in parallel) it is normal with this equipment?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
